### DISCUSSION TITLE
welcome to Upryzing's Discussions tab!

### DISCUSSION THREAD

**Discussion by Rexogamer** [upryzing/meta#2] [Announcements]
heya! welcome to our Discussions tab. we'll be using this area to discuss and plan new features that require **cross-repository implementation**, as well as any breaking changes. some examples:

- "add a timezone field to profiles": should be opened here as a discussion, as it requires implementation on the frontend and backend + in these cases further discussion would be useful (e.g. "what format should they use?")
- "make seasonal effects toggleable": should be fine as an issue on the relevant frontend repository, as it's a smaller change that only requires work in one area
- "the "Leave Space" button isn't working": should be opened as an issue on the relevant repository - bugs are (usually) confined to one or two areas, so they don't need a bigger discussion 
- "drop the `/foo/bar` API route": best as a discussion - breaking changes will usually require wider input/notice (this also applies to frontend breaking changes, e.g. "raise the Android app's minimum OS version to 9")
- "add a new API route that returns a list of the spaces you're in": should be fine as an issue on the backend repository, assuming it's about providing easier/better access to existing features and not introducing entirely new ones

these guidelines may be amended over time as needed - if you're unsure, feel free to ask here for help. thanks!

**Comment by sussycatgirl** [comment]
https://github.com/user-attachments/assets/7fada03a-b2f5-460c-be00-cca01c18e656
